source/partials/release_notes/_release-17-8-0.md.erb

```diff
@@ -0,0 +1,78 @@
+<h4>Bundled Plugins</h4>
+
+With this release, GoCD provides support for [Pipleine as code](https://docs.gocd.org/current/advanced_usage/pipelines_as_code.html) through [JSON config repo plugin](https://github.com/tomzo/gocd-json-config-plugin) and the [YAML config repo plugin](https://github.com/tomzo/gocd-yaml-config-plugin). These plugins maintained by our core contributor [Tomasz Sętkowski](https://github.com/tomzo) will be bundled by default.
+
+
+<h4>Improvements</h4>
+
+* <%= link_to_commit '1ccb0294265b4baea091c9068d7813fdde5009b4', 'Ensure order of auth_config in config xml is used to derive the order in which plugins are used for authentication.', 3608 %>
+* <%= link_to_commit '36d546e680aa43c31bf29d2a586ca31e57739bde', 'Remove tfs-impl-10' %>
+* <%= link_to_commit 'd7c4d167fc10101b03af4cd6676c806cef244747', 'Paused pipelines shows the pause reason on the history page.', 2784 %>
+* <%= link_to_commit '52ac924aad78ad2795c088199eb4c70f14b8205a', 'Save agent hostname and ip address as part of build history. This information will be displayed on the job details page for all kinds of agents.', 3368 %>
+
+<h4>Docker Images</h4>
+
+* Some improvements have been made to reduce the size of the docker GoCD [server image](https://hub.docker.com/r/gocd/gocd-server/). Please check the changelog [here](https://github.com/gocd/docker-gocd-server/blob/master/CHANGELOG.md).
+* Some improvements have been made to create slimmer docker GoCD agent images. Please check the changelog [here](https://github.com/gocd/docker-gocd-agent/blob/master/CHANGELOG.md).
+We provide support for the docker images for GoCD agents across the following operating systems - Centos [6](https://hub.docker.com/r/gocd/gocd-agent-centos-6/) & [7](https://hub.docker.com/r/gocd/gocd-agent-centos-7/) , Debian [7](https://hub.docker.com/r/gocd/gocd-agent-debian-7/) & [8](https://hub.docker.com/r/gocd/gocd-agent-debian-8/), Ubuntu [12.04](https://hub.docker.com/r/gocd/gocd-agent-ubuntu-12.04/), [14.04](https://hub.docker.com/r/gocd/gocd-agent-ubuntu-14.04/) & [16.04](https://hub.docker.com/r/gocd/gocd-agent-ubuntu-16.04/)
+
```

Comment:
Amazon ECS Elastic Agent Plugin

ThoughtWorks now offers a new commercial plugin which allows for flexible use of EC2 instances (with support for ECS Docker containers). This helps in optimizing utilization and reducing the cost of your agent infrastructure. For more information about the plugin, click here.